src/hdtSkyrimSystem.cpp

```
@@ -158,32 +158,32 @@ namespace hdt
 		auto bone = static_cast<SkyrimBone*>(m_mesh->findBone(getRenamedBone(name)));
 		if (bone) {
 			return bone;
 		}
 
-		logger::warn("Bone {} used before being created, trying to create it with current default values", name->cstr());
+		logger::warn("Bone {} used before being created, trying to create it with current default values", name.c_str());
 		return createBoneFromNodeName(name);
 	}
 
-	IDStr SkyrimSystemCreator::getRenamedBone(IDStr name)
+	const RE::BSFixedString& SkyrimSystemCreator::getRenamedBone(const RE::BSFixedString& name)
```

Clang (14.0.6)

[warning] use a trailing return type for this function

(modernize-use-trailing-return-type)